circle.yml

```diff
@@ -1,20 +1,116 @@
-dependencies:
-  override:
-    - curl -L https://atom.io/download/deb -o atom-amd64.deb
-    - sudo dpkg --install atom-amd64.deb || true
-    - sudo apt-get update
-    - sudo apt-get -f install
-    - node --version
-    - npm --version
-    - atom --version
-    - rm -rf node_modules; npm install
-    - npm run compile
+version: 2
 
-test:
-  override:
-    - npm run lint
-    - npm test
+defaults: &defaults
+  working_directory: /tmp/project
+  docker:
+    - image: arcanemagus/atom-docker-ci:stable
+  steps:
+    # Restore project state
+    - attach_workspace:
+        at: /tmp
+    - run:
```

Comment:
I'm a bit surprised this didn't cause a build error, but this step is only necessary if you need to install additional OS level packages. Since that isn't necessary here it should just be removed.